packages/expo-cli/src/prompts.ts

```diff
@@ -0,0 +1,34 @@
+import program from 'commander';
+import prompts, { Choice, PromptObject, PromptType } from 'prompts';
+
+import CommandError from './CommandError';
+
+export { PromptType, PromptObject };
+
+type CliPrompts = PromptObject | PromptObject[];
+
+export default function prompt(
+  cliPrompts: CliPrompts,
+  { nonInteractiveHelp }: { nonInteractiveHelp?: string } = {}
+) {
+  const nPrompts = Array.isArray(cliPrompts) ? cliPrompts.length : 1;
+  if (program.nonInteractive && nPrompts !== 0) {
+    let message = `Input is required, but Expo CLI is in non-interactive mode.\n`;
+    if (nonInteractiveHelp) {
+      message += nonInteractiveHelp;
+    } else {
+      const question: any = Array.isArray(cliPrompts) ? cliPrompts[0] : cliPrompts;
+      message += `Required input:\n${(question.message || '').trim().replace(/^/gm, '> ')}`;
+    }
+    throw new CommandError('NON_INTERACTIVE', message);
+  }
+  return prompts(cliPrompts, {
+    onCancel() {
+      throw new CommandError('ABORTED');
+    },
+  });
+}
+
+// todo: replace this workaround, its still selectable by the cursor
+// see: https://github.com/terkelg/prompts/issues/254
+prompt.separator = (title: string): Choice => ({ title, disable: true, value: undefined });
```

Comment:
As stated above, there is no separator just yet. They are working on a [plugin API to make this happen](https://github.com/terkelg/prompts/issues/254). For now, I'd figure this kinda brings the same functionality but the cursor will select it. (If you select it, a warning appears with "this answer is disabled")

Reply:
Nice trick 😄